build/migrations/21_Player_money.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE players ADD wallet int;
```

Comment:
Il faut ajouter une contrainte NOT NULL et un DEFAULT à 0 pour que l'intégrité des données soit assurée :)